### PR TITLE
Refactor function/class names to follow style guide

### PR DESCRIPTION
**Pull request by TaWeiTu**

### CODE REVIEW

File: src/testsuite.cpp [hunk 1]
```diff
@@ -275,14 +275,16 @@ int compile(const submission& target, int boxid, int spBoxid) {
   } else if (target.lang == "python2") {
     args = {"/usr/bin/env", "python2", "-m", "py_compile", "main.py"};
   } else if (target.lang == "python3") {
+    args = {"/usr/bin/env", "python3.7", "-c",
```
Comment (oToToT) [inline src/testsuite.cpp]: I think here is a mistake when merging commits.

Comment (TaWeiTu) [reply src/testsuite.cpp]: Fixed at 0ecef8c

File: src/utils.h [hunk 2]
```diff
@@ -160,80 +160,61 @@ std::string TdMeta(int prob, int td);
...
-      case CE:
-        return "CE";
-      case CO:
-        return "CO";
-      case ER:
-        return "ER";
-      default:
-        return "";
-    }
+inline const char* VerdictToStr(int verdict) {
```
Comment (oToToT) [inline src/utils.h]: is it better to change `int verdict` to `RESULTS verdict`?

Comment (TaWeiTu) [reply src/utils.h]: Fixed at 1f73053.

**Review by oToToT — APPROVED** [review]
LGTM!

**Pull request merged**